src/components/core/table/components/BaseTable.js

```diff
@@ -0,0 +1,99 @@
+/* @flow */
```

Comment:
We don't currently have a `core` directory under components in elements, can this live under just `components`? Also how does this table component differ from the existing `src/components/table` component? Maybe this can be `VirtualizedTable`?

Reply:
File has been moved to `src/components/base-table/`.

Right now `src/components/table` is just an HTML table, not a React Virtualized table.  `BaseTable` is just acting as a wrapper to create a pure component of a React Virtualized table with logic for striping (if that's ever a desired feature).  I could put this `BaseTable` code directly in `GridView`, but that may lower reusability if a different element wants to use the same logic as `BaseTable`.